pkgs/by-name/go/golem/package.nix

```
@@ -0,0 +1,70 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  protobuf,
+  redis,
+  fontconfig,
+  versionCheckHook,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "golem";
+  version = "1.0.23";
+
+  src = fetchFromGitHub {
+    owner = "golemcloud";
+    repo = "golem";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-ZvqlWavntlzQLUuTZG+pCeEZTT+DTP+QjiRgXu/9TvQ=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    protobuf
+  ];
+
+  buildInputs = [
+    fontconfig
+    (lib.getDev openssl)
+  ];
+
+  # Required for golem-wasm-rpc's build.rs to find the required protobuf files
+  # https://github.com/golemcloud/wasm-rpc/blob/v1.0.6/wasm-rpc/build.rs#L7
+  GOLEM_WASM_AST_ROOT = "/build/cargo-vendor-dir/golem-wasm-ast-1.0.1";
+  # Required for golem-examples's build.rs to find the required Wasm Interface Type (WIT) files
+  # https://github.com/golemcloud/golem-examples/blob/v1.0.6/build.rs#L9
+  GOLEM_WIT_ROOT = "/build/cargo-vendor-dir/golem-wit-1.0.1";
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "cranelift-bforest-0.108.1" = "sha256-WVRj6J7yXLFOsud9qKugmYja0Pe7AqZ0O2jgkOtHRg8=";
+    };
+  };
+
+  # Tests are failing in the sandbox because of some redis integration tests
+  doCheck = false;
+  checkInputs = [ redis ];
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = [ "${placeholder "out"}/bin/golem-cli" ];
+  versionCheckProgramArg = [ "--version" ];
+  doInstallCheck = true;
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "Golem is an open source durable computing platform that makes it easy to build and deploy highly reliable distributed systems.";
```

```suggestion
    description = "open source durable computing platform that makes it easy to build and deploy highly reliable distributed systems";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description